modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -110,7 +110,7 @@ ip=10.10.10.3::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none
 
 a|Optional: You can configure routes to additional networks by setting an `rd.route=` value.
 
-If the additional network gateway is different from the primary network gateway, the default gateway must be the primary network gateway.
+When you configure one or multiple networks, one default gateway is required. If the additional network gateway is different from the primary network gateway, the default gateway must be the primary network gateway.
```

Comment:
@miabbott I got feedback from the IBM Z QE @philc7777 that this change is also valid for x86. Can you please verify and ack. Thanks!

Reply:
Yeah, this makes sense to me.  :+1:

Reply:
As the Power QE, LGTM